src/PassportServiceProvider.php

```diff
@@ -91,6 +91,7 @@ public function register()
         $this->mergeConfigFrom(__DIR__.'/../config/passport.php', 'passport');
 
         Passport::setClientUuids($this->app->make(Config::class)->get('passport.client_uuids', false));
+        Passport::useTokenRepository();
```

Comment:
Register the token repository instead of using a static method. Place the registration in a separate method like the methods below.